app/models/spree/user.rb

```diff
@@ -148,10 +148,6 @@ def last_incomplete_spree_order
       spree_orders.incomplete.where(created_by_id: id).order('created_at DESC').first
     end
 
-    def flipper_id
```

Comment:
There is a test related to [flipper_id](https://github.com/openfoodfoundation/openfoodnetwork/blob/5dd27378118ca9626ceac0f71da972971d47a156/spec/models/spree/user_spec.rb#L203)
Do we keep it?

Reply:
I think it's a great idea, since this id is already used in some server, and it's fine to check it did not change (and then we don't need any data migration)

Reply:
@jibees 
I did a quick search for `flipper_id`, and the only two occurrences were on the spec file.
![image](https://user-images.githubusercontent.com/16113997/223368969-62500187-4051-4ee4-92d5-5ac93edc2afe.png)
is it used under another name?

Reply:
Good find. I agree with both of you. It's good that we kept the spec to check that the flipper id didn't change. But now that we verified that the flipper id is the same as before, we can remove the spec and trust flipper to tell us in release notes if it needs changing. Hm. :thinking: No, I'm changing my mind. Let's keep the spec in case it ever changes and we miss it. It's better to find out in the spec than to have all features for particular users deactivated by accident.

While we don't use the id in the code, we enter it manually in the flipper-ui as admin to activate a feature for a particular user. So we do rely on it. Maybe a better test case would be to enable a feature for an id and then make sure that it's recognised as enabled for a user with that id.

I would also like if we didn't have to enter the id but could select a user in the UI but that would be a new feature within Flipper. I don't think we have capacity for that right now.

To summarize, I think that we should keep the spec as is for now until we find a bug that needs a different spec.

Reply:
> Let's keep the spec in case it ever changes and we miss it. It's better to find out in the spec than to have all features for particular users deactivated by accident.

👍